cluster-service/update_instance_types.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+
+import os
+import json
+import yaml
+from yaml.resolver import BaseResolver
+
+class AsLiteral(str):
+  pass
+
+def represent_literal(dumper, data):
+  return dumper.represent_scalar(BaseResolver.DEFAULT_SCALAR_TAG, data, style="|")
+
+yaml.add_representer(AsLiteral, represent_literal)
+
+region = "westus3"
+
+output_helm_configmap_path = "deploy/templates/cloud-resources-config.configmap.yaml"
+with open(output_helm_configmap_path) as f:
+    cm_content = yaml.safe_load(f)
+
+# shelling out here to avoid importing all kinds of Azure libraries via pip
+json_out = os.popen(f"az vm list-sizes --location \"{region}\"").read()
```

Comment:
can we document that an active az session is a requirement to running this script.

Reply:
added a comment header and argparse, to print out a proper help